docs/integration_tests.md

```diff
@@ -1,119 +1,94 @@
+* Running Integration Tests with default context for a specific branch (code changes ouside of `/test` directory)
+* Running Integration Tests with a context from a specific branch (code changes inside of `/test` directory)
 
-Starting and setting up K3s is easy:
+These two options can be also combined and currently only executions of all Integration Tests for all Testsuites is supported. The execution of the Tests is fairly easy:
+1. Navigate to the `Actions` tab in `keptn/keptn` repository (https://github.com/keptn/keptn)
+2. Choose `Integration Tests` from the left side menu
+3. Click on `Run Workflow`, where a dialog window will appear. Here you need to choose the context (`Use Workflow from`) of the tests you wish to use (`master` is default). You should use this `master` context unless you have not made any changes in the Integration Tests. Secondly you choose a branch, from which the CI build artifacts (docker images) should be used from. Here you mostly use the branch of the code you are currently working on and want to run Integration Tests for your code changes. Please be aware, that you need to wait for the docker images to be buit before you can execute the Integartion Tests.
 
-1. Download, install and run K3s (tested with versions 1.16 to 1.18):
-    ```console
-    curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=v1.18.3+k3s1 K3S_KUBECONFIG_MODE="644" sh -s - --no-deploy=traefik
-    ```
-1. Export the Kubernetes config using:
-    ```console
-    export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
-    ```  
-    Please read more about it [here](https://rancher.com/docs/k3s/latest/en/).
-1. Verify that everything has worked using `kubectl get nodes`
+## Run Integration Tests locally
+
+### Prepare your local environment to run integration tests
+
+When running integration tests locally, we recommend using either K3d or Minishift. Please use the set-up steps below to set up your local enviroment before installing 
+Keptn and running the Integration Tests.
 
-### Set-up steps for Minikube
+#### **Set-up steps for K3d (recommended on Linux)**
 
-In case you plan to use Minikube, the following steps are necessary to get Keptn running on Minikube:
+Starting and setting up K3d is easy:
 
-1. Download and install Minikube (versions 1.4 to 1.10 should work) from [https://github.com/kubernetes/minikube/releases](https://github.com/kubernetes/minikube/releases) for your operating system.
-1. Make sure you create a fresh Minikube cluster (using a VM):
+1. Download and install K3d (**Note**: please be aware you need to have Docker installed, more info here: https://k3d.io/v5.2.2/):
     ```console
-    minikube stop # optional
-    minikube delete # optional
-    minikube start [--vm-driver=...] --cpus 2 --memory 4096 # or: minikube start --cpus 6 --memory 12200
+    curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
```

Comment:
Not sure if it makes sense, but during the integration tests, we use k3d 4.4.8 which has different cluster creation arguments and is not fully compatible with k3d v5

Reply:
I recommend using k3d v5.2.2, as the arguments and all steps are prepared for this specific version and it works good on my local environment

Reply:
I agree with @odubajDT and suggest documenting the steps for v5.2.2
Eventually, also our CI will move to a more recent version 😉